Dockerfile.hosted-execution

```diff
@@ -0,0 +1,18 @@
+FROM node:18-alpine
+WORKDIR /demo-js/
+
+# Best practices
+COPY ./selenium/package.json /demo-js/selenium/
```

Comment:
It would be more beneficial to our users to see a Cypress example as very few people still use Selenium JS

Reply:
you mean the Selenium JS Bindings

Reply:
Cypress wouldn't be applicable. The benefit Photon provides is that by running Selenium/Appium tests in the Sauce cloud, it brings the tests closer to the VM's running Selenium and reduces the HTTP roundtrip times. 

Cypress wouldn't have that benefit. Running Cypress tests through saucectl would just add an unnecessary intermediary step 😉 😉

Reply:
Good point ;)